core/types/block_ext_test.go

```diff
@@ -106,3 +111,410 @@ func assertDifferentPointers[T any](t *testing.T, a *T, b any) {
 	// Note: no need to check `b` is of the same type as `a`, otherwise
 	// the memory address would be different as well.
 }
+
+func TestBlockExtraGetWith(t *testing.T) {
```

Comment:
This test and the `Body` equivalent aren't really necessary as (a) they're fundamental guarantees of `libevm` and (b) they're implicitly tested by the RLP round-trip tests.

Reply:
Sorry that was due to late-evening-i-was-tired change removing unused exported methods such as GetBlockExtra and SetBlockExtra. No longer needed indeed.

Reply:
Tests refactored and this is no longer needed, resolving this!